packages/examples/src/langium/langium-dsl/config/extendedConfig.ts

```diff
@@ -10,17 +10,17 @@ import { useOpenEditorStub } from 'monaco-editor-wrapper/vscode/services';
 import { UserConfig } from 'monaco-editor-wrapper';
 import { BrowserMessageReader, BrowserMessageWriter } from 'vscode-languageclient/browser.js';
 import { loadLangiumWorker } from '../wrapperLangium.js';
-import { getTextContent } from '../../../common/client/app-utils.js';
+import langiumLanguageConfig from './langium.configuration.json' assert { type: 'json' };
+import langiumTextmateGrammar from './langium.tmLanguage.json' assert { type: 'json' };
+// @ts-expect-error otherwise the vite notation leads to a compile error
+import text from '../content/example.langium?raw';
```

Comment:
You can create the proper .d.ts instead of ignoring errors

Reply:
something like
```typescript
declare module '*?raw' {
  const content: string
  export default content
}
```

Reply:
@CGNonofr thank you fixed below. ⬇️